app/views/shared/_ie_fallbacks.html.slim

```diff
@@ -0,0 +1,5 @@
+- nonce = content_security_policy_script_nonce
+
+<script nonce="#{nonce}" src='https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js'></script>
```

Comment:
If they never got loaded, then we never used them....so definitely seems better to remove? Otherwise it's just cruft?